packages/imodel-content-tree/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@bentley/imodel-content-tree",
+  "version": "0.0.1",
+  "description": "React tree component that shows iModel content",
+  "keywords": [
+    "Bentley",
+    "iModel.js",
+    "iModel Content",
+    "Tree"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/imodel-content-tree"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/imodel-content-tree.js",
+  "typings": "lib/imodel-content-tree",
+  "scripts": {
+    "build": "tsc 1>&2 && npm run copy:assets",
+    "clean": "rimraf lib",
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "lint": "eslint ./src/**/*.{ts,tsx} 1>&2",
+    "test": ""
+  },
+  "dependencies": {
+    "classnames": "^2.2.5"
+  },
+  "devDependencies": {
+    "@bentley/bentleyjs-core": "^2.5.1",
```

Comment:
I could be mistaken, but these dev deps should probably be pinned down

Reply:
I don't know if this is necessary, I actually meant to talk to everyone else about this. I think we could move to using `^` everywhere since that's what our peerDeps state as well.

Reply:
Resolving this. @aruniverse we can chat separately about what to do here.